### DISCUSSION TITLE
Check variable value with run a script

### DISCUSSION THREAD

**Discussion by sorvict** [synfron/ReshaperForBurp#33] [Q&A] [answered]
Hi! I need to create a script to check if current value of one of the variables has symbol @ and if result is negative change the value of this variable to empty. Can someone help with that please?

**Comment by ddwightx** [comment, marked as answer]
You don't actually need a Script to do this. You could just use the built-in Rule operations. 

However, if you rather use a Then Run Script script, it would look like this:

```
const variableValue = Reshaper.variables.getGlobalVariable("nameOfTheVariable");
if (variableValue && !variableValue.includes("@")) {
  Reshaper.variables.setGlobalVariable("nameOfTheVariable", "");
}
```
The above assumes you are using a global variable. However if you are using an event variable, you would use `getEventVariable` and `setEventVariable` instead of `getGlobalVariable` and `setGlobalVariable`.

**Reply by sorvict** [reply]
Thank you for your help here! Appreciate this.
I believe I have interesting use case how I use Reshaper)
You can check see short demo here - https://www.loom.com/share/18e9b561b195413a9f77e9ff9b8a0084
